docs/querying/examples.md

```diff
@@ -38,6 +38,16 @@ To select all HTTP status codes except 4xx ones, you could run:
 
     http_requests_total{status!~"4.."}
 
+## Subquery
+
+This query returns 5-minute rate of `http_requests_total` metric for the past 30 minutes, at a resolution of 1 minute.
+
+    rate(http_requests_total[5m])[30m:1m]
+
+This is an example of nested subquery. The subquery for the rate uses default resolution. Note that using subqueries unnecessarily is unwise.
+
+    max_over_time(rate(http_requests_total[30s:5s])[10m:])
```

Comment:
This isn't a good example, you should use a range vector selector with rate.

Reply:
I thought so, will update it.

Reply:
`max_over_time(rate(rate(distance_covered_total[5s])[30s:5s])[10m:])` - does this look sane? Looks like max acceleration over time.

Reply:
A rate of a rate doesn't make sense, as rate can only be applied to a counter and the output of rate is a gauge. A deriv of a rate would make sense though.

Reply:
So does `max_over_time(deriv(rate(distance_covered_total[5s])[30s:5s])[10m:])` look fine for the doc?

Reply:
Yes, that's fine.